cmd/create/cluster/cmd.go

```diff
@@ -2347,82 +2333,34 @@ func validateOperatorRolesAvailabilityUnderUserAwsAccount(awsClient aws.Client,
 	return nil
 }
 
-func handleByoOidcOptions(r *rosa.Runtime, cmd *cobra.Command, isSTS bool) (bool, string, string) {
-	oidcEndpointUrl := args.oidcEndpointUrl
-	oidcPrivateKeySecretArn := args.oidcPrivateKeySecretArn
-	isByoOidcSet := false
+func handleOidcConfigOptions(r *rosa.Runtime, cmd *cobra.Command, isSTS bool) *v1.OidcConfig {
+	oidcConfigId := args.oidcConfigId
 	if isSTS {
-		// Support byo oidc config
-		if oidcEndpointUrl != "" && interactive.Enabled() {
-			oidcEndpointUrl, err := interactive.GetString(
+		if oidcConfigId == "" && interactive.Enabled() {
+			_oidcConfigId, err := interactive.GetString(
 				interactive.Input{
-					Question:   "OIDC Endpoint URL",
-					Help:       cmd.Flags().Lookup(OidcEndpointUrlFlag).Usage,
-					Required:   false,
-					Default:    oidcEndpointUrl,
-					Validators: []interactive.Validator{interactive.IsURL},
+					Question: "OIDC Config ID",
+					Help:     cmd.Flags().Lookup(oidcConfigId).Usage,
```

Comment:
@gdbranco if the `oidcConfigId` isn't provided by the user, we get panic here.
The `Lookup` function returns **nil**.
![image](https://user-images.githubusercontent.com/57869309/227773168-9e9c7bdc-cc5f-48ee-9f32-4f57591c9042.png)

Is this flow still optional for Hypershift clusters?